src/Nethermind/Nethermind.Core.Test/Blockchain/TestBlockchain.cs

```diff
@@ -51,7 +52,7 @@ namespace Nethermind.Core.Test.Blockchain
 {
     public class TestBlockchain : IDisposable
     {
-        public const int DefaultTimeout = 2000;
+        public const int DefaultTimeout = 4000;
```

Comment:
you want to push this?

Reply:
yes its for tests and good for dot cover